lib/views/loading_screen.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+import 'home.dart';
+import '../models/repository/genre_repository.dart';
+import '../models/repository/movie_repository.dart';
+
+class LoadingScreen extends StatefulWidget {
```

Comment:
You need this loading screen? You can do the same on HomePage, using the waiting of the bloc

Reply:
done